docs/DeveloperGuide.md

```diff
@@ -94,12 +66,14 @@ It is also responsible for displaying the response from PatientManager to the sc
 
 The Sequence Diagram below shows how the components interact with each other for the scenario where the user issues the command `add S1234567D`.
 
-{TO BE ADDED}
+<img src="./images/sequence_diagram.png" width="600">
```

Comment:
Would it be better if to use markdown's inbuilt tag instead?
Like this one for [our user guide](https://github.com/AY2021S2-CS2113T-W09-4/tp/blame/master/docs/UserGuide.md#L38)!

Reply:
I was considering that, but it seems that github flavoured markdown doesn't support resizing images from within the tag (see https://gist.github.com/uupaa/f77d2bcf4dc7a294d109). It seems like I have to fall back on `<img>` tags to control the image size.